spec/components/schemas/content/voice/call.ts

```diff
@@ -0,0 +1,60 @@
+import { SchemaObject } from 'openapi3-ts';
+import { ref as baseRef } from './../base';
+import { ref as ttsRef } from '../tts';
+import { ref as audioFileRef } from '../audiofile';
+import { createComponentRef } from '../../../../../utils/ref';
+
+const text: SchemaObject = {
+  type: 'object',
+  allOf: [{
+    $ref: baseRef,
+  }, {
+    type: 'object',
+    properties: {
+      type: {
+        type: 'string',
+        example: 'call',
+      },
+      audio: {
+        description: 'An array of audio actions',
+        type: 'array',
+        items: {
+          anyOf: [{
+            $ref: ttsRef,
+          }, {
+            $ref: audioFileRef,
+          }],
+          discriminator: {
+            propertyName: 'type',
+            mapping: {
+              tts: ttsRef,
+              file: audioFileRef,
+            },
+          },
+        },
+      },
+      transferTo: {
+        description: 'Transfer or call to another number',
```

Comment:
Acho que poderia complementar com algo tipo,
"after the (optional) audio message has been played" (assumindo que alteraremos a outra descrição).